planner/core/cache.go

```diff
@@ -121,12 +122,35 @@ func NewPSTMTPlanCacheKey(sessionVars *variable.SessionVars, pstmtID uint32, sch
 
 // PSTMTPlanCacheValue stores the cached Statement and StmtNode.
 type PSTMTPlanCacheValue struct {
-	Plan Plan
+	Plan   Plan
+	Chunks map[string]*chunk.Chunk
+}
+
+// Delete implements Value interface.
+func (value *PSTMTPlanCacheValue) Delete() {
+	value.Chunks = make(map[string]*chunk.Chunk)
```

Comment:
can we set it be `nil`?

Reply:
The updated PR should address it. Thank you!